packages/integrations/src/awslambda.ts

```diff
@@ -0,0 +1,230 @@
+import * as Sentry from '@sentry/browser';
+import { EventProcessor, Hub, Integration, Scope } from '@sentry/types';
+
+/**
+ * NodeJS integration
+ *
+ * Provides a mechanism for NodeJS
+ * that raises an exception for handled, unhandled, timeout and similar times of error
+ * and captures the same in Sentry Dashboard
+ */
+export class AWSLambda implements Integration {
+  /**
+   * @inheritDoc
+   */
+  public static id: string = 'AWSLambda';
+
+  /**
+   * @inheritDoc
+   */
+  public name: string = AWSLambda.id;
+
+  /**
+   * context.
+   */
+  private _awsContext: {
+    getRemainingTimeInMillis: () => number;
+    callbackWaitsForEmptyEventLoop: boolean;
+    awsRequestId: string;
+    functionName: string;
+    functionVersion: string;
+    invokedFunctionArn: string;
+    logGroupName: string;
+    logStreamName: string;
+  };
+
+  /**
+   * timeout flag.
+   */
+  private _timeoutWarning?: boolean = false;
+
+  /**
+   * @inheritDoc
+   */
+
+  /**
+   * flush time in milliseconds to set time for flush.
+   */
+  private _flushTime?: number;
+
+  public constructor(options: { context?: any; timeoutWarning?: boolean; flushTime?: number } = {}) {
+    this._awsContext = options.context;
+    this._timeoutWarning = options.timeoutWarning;
+    this._flushTime = options.flushTime;
+  }
+
+  /**
+   * @inheritDoc
+   */
+  public setupOnce(_: (callback: EventProcessor) => void, getCurrentHub: () => Hub): void {
+    const lambdaBootstrap: any = require.main;
+
+    /** configured time to timeout error and calculate execution time */
+    const configuredTimeInMilliseconds = this._awsContext.getRemainingTimeInMillis();
+
+    if (!this._awsContext && !lambdaBootstrap) {
+      return;
+    }
+    const processEnv: any = process.env;
+    /** rapid runtime instance */
+    const rapidRuntime = lambdaBootstrap.children[0].exports;
```

Comment:
```suggestion
let rapidRuntime;

 try {
   rapidRuntime = lambdaBootstrap.children[0].exports;
 } catch (_oO) {
   rapidRuntime = () => {}
}
```

We need to code defensively everywhere, as we are opt-in SDK. We should never break anyone's else code.

Reply:
Made these changes but instead of passing blank function **rapidRuntime = () => {}**, we pass the blank object to the rapitRuntime to avoid Lint error  **rapidRuntime =  {}**.